src/main/resources/site/parts/husleieCalculator/husleieCalculator.jsx

```diff
@@ -396,6 +403,12 @@ function HusleieCalculator(props) {
   }
 
   function calculatorResult() {
+    const resultScreenReaderText = props.phrases.husleieNewRentResult
+      .replace('{0}', endValue)
```

Comment:
erstatte punktum med komma for endValue og change hvis man er på norsk kalkulator.
`.replace('{0}', language === 'en' ? endValue : endValue.replace(/\./g, ','))
      .replace('{1}', language === 'en' ? change : change.replace(/\./g, ','))`

Reply:
implemented changes :)